web/src/pages/eventlist.js

```diff
@@ -1,75 +1,100 @@
-import React, {Component} from 'react';
-import EventFilters from '../components/eventslist-component/EventFilters';
-import EventCards from '../components/eventslist-component/EventCards';
-import Grid from '@material-ui/core/Grid';
-
+import EventCardsQuery from '../components/event-cards-query';
+import EventlistFilters from '../components/eventlist-filters';
 import {gql} from 'apollo-boost';
-import { Query } from "react-apollo";
-
-/* Query requests event list to server */
-const EVENT_LIST_REQUEST_QUERY = gql`
-    query{
-        events(pageSize: 9){
-            cursor,
-            events{
-                id,
-                title,
-                description
-            }
-        }
-    }`;
+import Grid from '@material-ui/core/Grid';
+import {Query} from "react-apollo";
+import React, {Component} from 'react';
 
 class EventList extends Component {
-  constructor(props) {
-    // get area selection from props
-    super(props);
 
+  constructor() {
+    super();
+    this.state = {
+      isTagNames : false,
+      allTags : [],
+      selectedTagIds : [],
+      eventListPageSize: 9,
+      currentCursor: 0
+    }
```

Comment:
don't leave space between variable name and ':' 
change like this
variable: value